addons/core/script_macros.hpp

```diff
@@ -131,3 +131,5 @@
 #define PROFCONTEXT_LOGTRAP(VAR,FUNC)
 
 #endif
+
+#define LOCALIZE_CHANNEL(x,y) y call _fnc_localize##x##Channel
```

Comment:
@dedmen 
Ok, I add macros here.

Reply:
Looks correct. But please put it into fnc_initKeybinds as that macro is not used anywhere else.

Reply:
Ok